resources/sdk/clisdkclient/extensions/services/commandservice.go

```diff
@@ -288,6 +288,7 @@ func reAuthenticateIfNecessary(config config.Configuration, err error) error {
 }
 
 func (c *commandService) DetermineAction(httpMethod string, uri string, flags *pflag.FlagSet) func(retryConfiguration *retry.RetryConfiguration) (string, error) {
+	logger.InitLogger(c.cmd)
```

Comment:
Initialising the logger here to give it access to the configuration via the "profile" flag.